Client/src/Components/CreateMaintenanceWindow/index.jsx

```diff
@@ -0,0 +1,152 @@
+import { Box, Stack, Typography } from "@mui/material";
```

Comment:
This is a Page, not a Component, so it should be loctaed at `src/Pages/Maintenance/CreateMaintenanceWindow/index.jsx`